README.md

```diff
@@ -1,65 +1,86 @@
 
-![Example of live coding an app in Streamlit|635x380](https://raw.githubusercontent.com/streamlit/docs/main/public/images/Streamlit_overview.gif)
 
 ## Installation
 
+Open a terminal and run:
 ```bash
-pip install streamlit
-streamlit hello
+$ pip install streamlit 
+$ streamlit hello
 ```
 
-Streamlit can also be installed in a virtual environment on [Windows](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-windows), [Mac](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux), and [Linux](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux).
+This should open our sweet _Streamlit Hello_ app right in your browser.
+
+<img src="https://media.cleanshot.cloud/media/55064/9RAuEZKubvrlrlPVua50xuhyPIiD8MBdvzi7Pa3s.gif?Expires=1674790062&Signature=fqIRLW0tMXntC1vCrSq3yLPTtA58SRLiO1i0y~7YshYuOfQQokc0OI6uxWg6IBL8rw2gGnzA4ML-OdGWPgK6WSulyLR-wNbQAPiKX4LF0vXS9F6AmZ8V39Tze5Fo9TWfA3JZF3mnu7q6JrkssCk~6614xuugx9gojkOJMtsbFrOiM2kNLkje2OepBbxromYKVLLEQGwuPcFvUiA3Hb6ZuSHtNX~99wLu94PAKpXfPUf21sCt4vfmYItkl0z6BuHWYP5Zs5SlvPo8iXvdrb-dKL-~-kbTRjpcTFgWhFT1zg8iNZNCeR0Q~QvpS8rzfQPMMeIiBiRJFmTm4RC7~FRktA__&Key-Pair-Id=K269JMAT9ZF4GZ" alt="Streamlit Hello" width=500 href="none"></img>
+
+Read more on getting started with Streamlit in our [documentation](https://docs.streamlit.io/library/get-started).
 
-## A little example
+
+## API Reference
+
+#### A little example
 
 Streamlit makes it incredibly easy to build interactive apps:
 
 ```python
 import streamlit as st
-
-x = st.slider('Select a value')
-st.write(x, 'squared is', x * x)
+x = st.slider("Select a value")
+st.write(x, "squared is", x * x)
 ```
 
-<img src="https://raw.githubusercontent.com/streamlit/docs/main/public/images/simple_example.png"/>
+<img src="https://user-images.githubusercontent.com/7164864/214900507-1c89fc6b-e196-4f5c-890f-7e0be62d5d9d.png" width=300 alt="Little example"></img>
 
-## A bigger example
+#### Give me more!
 
-Streamlit's simple and focused API lets you build incredibly rich and powerful tools.  [This demo project](https://github.com/streamlit/demo-self-driving) lets you browse the entire [Udacity self-driving-car dataset](https://github.com/udacity/self-driving-car) and run inference in real-time using the [YOLO object detection net](https://pjreddie.com/darknet/yolo).
+Streamlit comes in with [a ton of additional handy elements](https://docs.streamlit.io/library/api-reference) to spice your data apps and delight your viewers. Some examples:
 
-![Final App Animation](https://raw.githubusercontent.com/streamlit/docs/main/public/images/complex_app_example.gif)
+<table border="0">
+   <tr>
+     <td><strong>Input widgets</strong><br><br><img src="https://docs.streamlit.io/images/api/date_input.jpg" height="150"></td>
```

Comment:
Is this table displayed correctly in PyPi? PyPi also uses README.md, so we should check if it displays correctly there too.
https://pypi.org/project/streamlit/

Reply:
Good point, although I guess most people look on GitHub and not on PyPI

Looking at github.com/huggingface/transformers README which displays a table using Markdown:

On GitHub
<img width="881" alt="image" src="https://user-images.githubusercontent.com/7164864/215112637-d92c8b4e-72f8-445d-86bf-dc52feeaa11d.png">

On PyPI
<img width="806" alt="image" src="https://user-images.githubusercontent.com/7164864/215112745-4986ad7b-84fb-45b8-b6e8-e617f8fb0a37.png">

And this seems to work as well when tables are given in HTML (see bokeh on [GitHub](https://github.com/bokeh/bokeh) and [PyPI](https://pypi.org/project/bokeh/)

Reply:
Can you check if [`streamlit-nighlty`](https://pypi.org/project/streamlit-nightly/#description) displays correctly after merging?